mlflow/projects/__init__.py

```diff
@@ -189,6 +194,22 @@ def _is_local_uri(uri):
     return not _GIT_URI_REGEX.match(uri)
 
 
+def _is_valid_branch_name(work_dir, branch):
+    """
+    Returns True if the branch exists in the Git project.
```

Comment:
For clarity (since `branch` isn't necessarily a branch name), maybe we can rename the `branch` argument in this method to `version`? And then change the docstring to:

```
Returns True if `version` is the name of a branch in a Git project.
``work_dir`` must be the working directory in a Git repo.
```

Reply:
Done.